stage #2.5/module #3 Spring Core/#5.2. Java-based configuration/content.md

```diff
@@ -0,0 +1,320 @@
+<h1 style="color: green">Java-based Container Configuration</h1>
```

Comment:
most likely package name should be started form #5.3, if you don't want to see 2 packages with the same number

Reply:
Changed, thanks.